src/main/java/com/iemr/hwc/data/foetalmonitor/FoetalMonitor.java

```diff
@@ -38,13 +38,13 @@
 
 @Entity
 @Table(name = "t_fetosensedata")
-public class Fetosense {
+public class FoetalMonitor {
 
 	@Id
 	@GeneratedValue(strategy = GenerationType.AUTO)
 	@Expose
 	@Column(name = "fetosenseID")
-	private Long partnerFetosenseId;
+	private Long partnerFoetalMonitorId;
```

Comment:
@swetatech24 @Anjana2310 Please change partnerFoetalMonitorId to foetalMonitorId